services/ppac/src/main/java/app/coronawarn/datadonation/services/ppac/ios/verification/apitoken/ApiTokenService.java

```diff
@@ -125,9 +127,14 @@ private void updatePerDeviceData(String deviceToken, String transactionId) {
         false,
         false);
     try {
-      iosDeviceApiClient.updatePerDeviceData(jwtProvider.generateJwt(), updateRequest);
+      ResponseEntity<String> response =
+          iosDeviceApiClient.updatePerDeviceData(jwtProvider.generateJwt(), updateRequest);
+      if (response.getStatusCodeValue() != HttpStatus.OK.value() || !"OK".equalsIgnoreCase(response.getBody())) {
```

Comment:
Maybe we can use cwa dev app to trigger a valid update request and see the actual response. Based on this we could craft maybe a more precise check?

Reply:
Done. Thanks for the support :-)